features/teacher_profile.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Management of the teacher profile
+  This feature contains registration, updating and deletion of a user account for teachers, including all profile data
+
+  Scenario: A new user registers themselves on the platform
```

Comment:
I think it should be either:
"A new teacher registers herself/himself on the platform"

or: 
"Teachers registering themselves on the platform"